### ISSUE TITLE
What's the loss used in training VAE?

### ISSUE THREAD

**Issue by JingyunLiang** [axium/Blind-Image-Deconvolution-using-Deep-Generative-Priors#2]
Hi, I didn't find the training code of VAEs. Although there is a testing code, I don't know which loss is used in training, nor do I find the loss in your paper. What's the loss  used in training a VAE for kernels and SVHN?

If possible, could you please provide the training code of VAE?

**Comment by fahadshamshad** [comment]
Hi, we have used standard VAE training loss and use the trained VAE in our deblurring algorithm. Feel free to open the issue again if you have any doubt.
